runtime/core/bin/frontend_t2s_main.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022, Binbin Zhang (binbzha@qq.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <glog/logging.h>
+#include <gflags/gflags.h>
+
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "frontend/t2s.h"
+
+DEFINE_string(t2s_file, "", "traditional to simplified dictionary");
```

Comment:
`t2s_file` is confusing (text to speech? traditional to simplified?). Why do not use `lexicon_file` here?